runtime/nls/cfre/cfrerr.nls

```diff
@@ -1615,7 +1458,7 @@ J9NLS_CFR_ERR_INCORRECT_FLAG_FOUND_ON_VALUE_CLASS.user_response=Contact the prov
 J9NLS_CFR_ERR_PERMITS_VALUE_ON_FINAL_CLASS=class has both ACC_FINAL and ACC_PERMITS_VALUE set
 # START NON-TRANSLATABLE
 J9NLS_CFR_ERR_PERMITS_VALUE_ON_FINAL_CLASS.explanation=Please consult the Java Virtual Machine Specification for a detailed explanation.
-J9NLS_CFR_ERR_PERMITS_VALUE_ON_FINAL_CLASS.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_PERMITS_VALUE_ON_FINAL_CLASS.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
```

Comment:
Can you apply this change throughout (i.e. change "verification or class loading-related" to "verification or classloading related"), please?

Reply:
done

Reply:
Not quite: I still see "class loading" where we want "classloading" (e.g. lines 1405, 1412, etc.).

Reply:
Sorry I thought it's just removing the hyphens. Should be fine now.